src/config_handling.py

```diff
@@ -30,7 +35,10 @@ def init_config(app):
     )
 
     for config in all_configs:
-        value = os.environ.get(config, get_default_for(config))
+        if config in file_configs:
+            value = get_config_from(app, config + "_FILE")
+        else:
+            value = os.environ.get(config, get_default_for(config))
         if not value:
             continue
```

Comment:
One thing, which came to my mind:
Try to read values for the credentials (user, password) for a database connection. If such values are empty, raise an exception instead of just continuing.

Reply:
No need to throw an exception here. There is code later, which checks all env vars, and throws an exception if s.th. is missing.

Reply:
Well, if this is the case, then the line 42 - 43 is not necessary and should be removed.

Reply:
Changed it.